### DISCUSSION TITLE
GEFS TC track error/spread verification

### DISCUSSION THREAD

**Discussion by JiayiPeng-NOAA** [dtcenter/METplus#1341] [Statistical Computation] [answered]
Does the MET-TC have the capability to verify GEFS ensemble mean TC track error and spread? The case numbers for track error calculation need to be matched with the case numbers for track spread calculation.

**Comment by JohnHalleyGotway** [comment]
Hello Jiayi,

Yes, the MET tool named TC-Pairs has the ability to compute a "consensus" track. The "consensus" track is really just an ensemble mean of the specified model tracks... where the location and intensity at each point is the mean of the locations and intensities of the members. So this assumes that you already have ATCF track data for each of the ensemble members, and the member names can be distinguished from one another. 

Please search for "consensus" in the [TC-Pairs Chapter](https://met.readthedocs.io/en/latest/Users_Guide/tc-pairs.html#tc-pairs-configuration-file) of the MET User's Guide.  That describes the required elements for each consensus dictionary array entry.

However, after generating one or more ensemble mean consensus tracks, they are just processed like any other track. The ensemble *SPREAD* is not directly evaluated anywhere. I'm wondering how you're anticipating the ensemble track and intensity spread should be evaluated?

I'll tag @KathrynNewman here, my colleague in the DTC who does tropical cyclone evaluation, to see if she has any input on the evaluation of spread.

**Comment by JiayiPeng-NOAA** [comment, marked as answer]
Hi John,
I do not know whether Met-TC has the capability to calculate the spread for
TC track and intensity (maximum wind speed).
Usually while we calculate the ensemble mean, we also calculate the
ensemble spread (either track or intensity).
When we verify track errors, we also calculate the statistics for spread
with its case-number matching with the track error case-number.
Thanks,
Jiayi

**Reply by JohnHalleyGotway** [reply]
@JiayiPeng-NOAA I read through the issues you posted for the NOAA METplus telecon for 1/24/22.

It's obviously not difficult to compute spread when computing the mean. But the issue is where/how to store the output. While the TC-Pairs tool can compute a consensus "mean" track and intensity forecast, it is not currently computing/storing the spread.

I recommend that we schedule a meeting to dive into these details, as well as METplus Discussion #1304 .

I'm hoping you can step through all of the details behind your comment:
   "When we verify track errors, we also calculate the statistics for spread with its case-number matching with the track error case-number."

We can also discuss #1304 and dive into the details of the format of the probability of genesis forecast you're trying to verify.
